README.md

```diff
@@ -0,0 +1,124 @@
+# Application Autoscaler
+
+The Application Autoscaler provides the capability to adjust the computation resources for Cloud Foundry applications through:
+
+* Dynamic scaling based on application performance metrics
+* Dynamic scaling based on custom metrics
+* Scheduled scaling based on time
+
+This repository contains the core Application Autoscaler source code, extracted and refactored from [app-autoscaler-release](https://github.com/cloudfoundry/app-autoscaler-release).
+
+## Architecture
+
+The Application Autoscaler consists of several microservices:
+
+| Component         | Description                                                                 |
+|-------------------|-----------------------------------------------------------------------------|
+| `api`             | Public-facing API server for policy management and scaling history          |
+| `servicebroker`   | Cloud Foundry service broker implementation                                 |
+| `scheduler`       | Manages scheduled scaling policies and triggers scaling actions             |
+| `eventgenerator`  | Evaluates scaling rules and generates scaling events based on metrics       |
+| `scalingengine`   | Executes scaling decisions by adjusting application instances               |
+| `metricsforwarder`| Forwards custom application metrics to the autoscaler                       |
+| `operator`        | Manages autoscaler operations and instance synchronization                  |
+
+## API Specifications
+
+OpenAPI specifications are available in the [`openapi/`](./openapi/) directory:
+
+* `application-metric-api.openapi.yaml` - Application metrics API
+* `custom-metrics-api.openapi.yaml` - Custom metrics submission API
+* `policy-api.openapi.yaml` - Scaling policy management API
+* `scaling-history-api.openapi.yaml` - Scaling history query API
+* `internal-scaling-history-api.openapi.yaml` - Internal scaling history API
+
+## Local Development
+
+### Prerequisites
+
+* [Go](https://golang.org/) 1.24.3 or later
+* [Docker](https://www.docker.com/products/docker-desktop/) to spin up required databases
+* [devbox](https://github.com/jetify-com/devbox) (optional but recommended) - starts a shell with all required tools
+* [Maven](https://maven.apache.org/) for building the Java-based scheduler component
+* [direnv](https://direnv.net/) (optional) to automatically configure the development environment
+
+### Make Targets
+
+| Target                                                                | Description                                                                |
```

Comment:
Can we just check one time if all of these make targets works and are really required to a operator/developer to use them